### ISSUE TITLE
ES6 string.includes

### ISSUE THREAD

**Issue by darsee** [sheepsteak/react-shallow-testutils#11]
Thanks for sharing this - it's really useful!

I would like to be able to use `findAllWithClass` but it currently fails with

```
TypeError: 'undefined' is not a function (evaluating 'className.includes('.')')
```

**Comment by sheepsteak** [comment]
Hi, thanks for finding this. This is because `String.includes` is an ES2016 feature that needs the `babel-polyfill`. I've always used `babel-polyfill` when using this module so I hadn't had this problem. I'll get this fixed now.

**Comment by sheepsteak** [comment]
@darsee #12 should fix this for you.

**Comment by darsee** [comment]
Confirmed. Thanks for the quick response!

**Comment by csilcock** [comment]
@sheepsteak - Thanks for addressing this issue in a recent version.

I'm trying to write some tests for a project that (unfortunately) has dependencies on React 0.13. I was wondering if you'd be able to patch version 0.4 to include this fix?

**Comment by sheepsteak** [comment]
@csilcock Good idea. I've just released 0.4.1 that contains the fix for the version using React 0.13.

**Comment by csilcock** [comment]
@sheepsteak thanks so much for the quick turnaround.
